packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -162,10 +170,12 @@ export class H5WasmApi extends DataProviderApi {
       }
 
       const plugin = PLUGINS_BY_FILTER_ID[filter.id];
-      if (!plugin || !this.getPlugin) {
-        throw new Error(
+      if (!plugin) {
+        // eslint-disable-next-line no-console
+        console.warn(
           `Compression filter ${filter.id} not supported (${filter.name})`,
         );
```

Comment:
> Arguable, I replaced those errors with as console warnings.

Hm why ? I thought the point of the PR was to throw errors for every kind of error to deal with them consistently ?

Reply:
We don't know that these cases will cause errors for sure (see `fcidecomp`). The goal is to let HDF5 throw the real errors now that we can catch them.

Reply:
Oh got it. If there is an error, then it should appear earlier then ?

Reply:
The warning will appear earlier, yes. If that's what you mean?

I'm not sure the first warning is needed, since we can just look at the filters in the metadata viewer and compare their IDs with the ones in the codebase to know whether we support them or not. The second warning is a bit more interesting and might help developers of consumer apps identify issues with their `getPlugin` implementations (even if unlikely).